### PR TITLE
Rdm 6410 functional tests validate case data

### PR DESCRIPTION
**Pull request by dharmendra8186**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-6410

### Change description ###
Implement Functional Tests for 'CCD Data Store Api :: Validate case data' operation

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/aat/resources/features/F-034/S-310.td.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    }
+  },
+
+  "expectedResponse": {
+    "_extends_": "Common_404_Response",
+    "body" : {
+      "exception" : "uk.gov.hmcts.ccd.endpoint.exceptions.ResourceNotFoundException",
+      "timestamp" : "[[ANY]]",
+      "status" : 404,
+      "error" : "Not Found",
```
Comment (RebeccaBaker) [inline src/aat/resources/features/F-034/S-310.td.json]: status and error are inherited from Common_404_Response

File: src/aat/resources/features/F-034/S-310.td.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      "message" : "Resource not found when getting case type version for abc because of 404 ",
```
Comment (RebeccaBaker) [inline src/aat/resources/features/F-034/S-310.td.json]: check PR build there is an assertion error with this message

File: src/aat/resources/features/F-034/S-314.td.json [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        "summary": "",
+        "description": ""
+      }
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
+    "headers": {
+      "Transfer-Encoding" : "chunked"
+    },
```
Comment (RebeccaBaker) [inline src/aat/resources/features/F-034/S-314.td.json]: check pr build, mis match in expected headers

File: src/aat/resources/features/F-034/S-310.td.json (hunk 1 above)
Comment (dharmendra8186) [reply src/aat/resources/features/F-034/S-310.td.json]: Done

**Pull request merged**